### DISCUSSION TITLE
Open ports directly to the web browser like new tab?

### DISCUSSION THREAD

**Discussion by MacPla** [seriousm4x/UpSnap#222] [Feature requests]
I think it's very powerful and convenient to be able to open ports directly to the web browser as a new tab.

We would have very great control over the machines, their services and their access from the browser.

Do you think it could be a good implementation?

Cheers,

**Comment by seriousm4x** [comment]
Will be available in the next release!

**Comment by taowei20** [comment]
I strongly suggest to add this function.
near every port config . add 2 input text , user can input open port command or close  port command.
like this:
ssh -o StrictHostKeyChecking=no  root@192.168.x.xx  -i /app/pb_data/id_rsa "ufw allow 80/tcp"
